backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  listingUtilities                      ListingUtilities?             @relation(fields: [utilitiesId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listingsApplicationMailingAddress     Address?                      @relation("application_mailing_address", fields: [applicationMailingAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listingsLeasingAgentAddress           Address?                      @relation("leasing_agent_address", fields: [leasingAgentAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listingFeatures                       ListingFeatures?              @relation(fields: [featuresId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  jurisdictions                         Jurisdictions?                @relation(fields: [jurisdictionId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listingsApplicationPickUpAddress      Address?                      @relation("application_pick_up_address", fields: [applicationPickUpAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  listingsBuildingAddress               Address?                      @relation("building_address", fields: [buildingAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  userAccounts                          UserAccounts[]
+  units                                 Units[]
+  unitGroup                             UnitGroup[]
+  userPreferences                       UserPreferences[]
+  unitsSummary                          UnitsSummary[]
+
+  @@index([jurisdictionId])
+}
+
+// Note: hold over from TypeORM 
+model Migrations {
+  id        Int    @id() @default(autoincrement())
+  timestamp BigInt
+  name      String @db.VarChar
+
+  @@map("migrations")
+}
+
+// Note: missing [untranslatedText, untranslatedOptOutText] virtual fields
+// Note: [options] formerly had array max length 64
+model MultiselectQuestions {
+  id                 String                                     @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt          DateTime                                   @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt          DateTime                                   @updatedAt @map("updated_at") @db.Timestamp(6)
+  text               String
+  subText            String?                                    @map("sub_text")
+  description        String?
+  links              Json?
+  options            Json?
+  optOutText         String?                                    @map("opt_out_text")
+  hideFromListing    Boolean?                                   @map("hide_from_listing")
+  applicationSection MultiselectQuestionsApplicationSectionEnum @map("application_section")
+  jurisdictions      Jurisdictions[]
+  listings           ListingMultiselectQuestions[]
+
+  @@map("multiselect_questions")
+}
+
+model PaperApplications {
```

Comment:
Okay my b I thought only a few had the plural issue I will just note it for the whole file 😅

Reply:
LOL dang I was actually hoping you had found all of them and we could make a convention through that, but either way yea we need to figure out whether these should be singular or plural